From beba27cc07da039b6d1ca26bca4f64bddf86f4bf Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Fri, 11 Oct 2013 13:22:14 +0800
Subject: [PATCH 7/7] ENGR00215174 fix build error with -llzo2 and -luuid on
 12.04 64bit ubuntu.

fix build error with -llzo2 and -luuid on 12.04 64bit ubuntu.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 mkfs.ubifs/Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkfs.ubifs/Android.mk b/mkfs.ubifs/Android.mk
index 8fc1536..b4a4984 100644
--- a/mkfs.ubifs/Android.mk
+++ b/mkfs.ubifs/Android.mk
@@ -18,9 +18,9 @@ LOCAL_CC := /usr/bin/gcc
 LOCAL_CXX := /usr/bin/g++
 LOCAL_CFLAGS = -O2 -Wall
 LOCAL_CFLAGS+= -Wpointer-arith -Wwrite-strings -Wstrict-prototypes
-LOCAL_CFLAGS+= -Wmissing-prototypes -Winline
+LOCAL_CFLAGS+= -Wmissing-prototypes -Winline -m64
 LOCAL_LDFLAGS += -L/usr/lib/x86_64-linux-gnu
-LOCAL_LDLIBS:= -lz -llzo2 -lm -luuid
+LOCAL_LDLIBS:= -lz -llzo2 -lm -luuid -m64
 
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/
 LOCAL_C_INCLUDES += /usr/include/
-- 
1.8.0

